src/amo/components/InstallButtonWrapper/index.js

```diff
@@ -92,49 +93,65 @@ export const InstallButtonWrapperBase = (props: InternalProps) => {
 
   const showDownloadLink = !isCompatible && installURL;
 
-  return (
-    addon && (
+  if (latest === false) {
+    return (
       <div
```

Comment:
@bobsilverberg Did you mean this div that is there twice? Because the code for the button i.e. `AMInstallButton` is only written at one place.

Reply:
The code to display an download link is included twice in this file. Once here, and once again in the section where `InstallButtonWrapper-download` is defined. My suggestion is that we have code to display a download link only defined once, perhaps in a function, and then that function can be called in each place, rather than having to duplicate the code. Can you give that a try?

Reply:
@bobsilverberg Both of the links have different CSS classes and i18 text. I can define a separate function but that'll essentially be a lot more lines as compared to what we have currently. Also that function will just take care of the `<a> </a>` tag. If you want I can go ahead and define it but I think what we have currently works better. Let me know what you think about this.